docs/pages/get-started/installation.md

```diff
@@ -2,52 +2,51 @@
 title: Installation
 ---
 
-There are two tools that you need to develop apps with Expo: a local development tool and a mobile client to open your app.
+import TerminalBlock from '~/components/plugins/TerminalBlock';
 
-## 1. Local development tool: Expo CLI
+There are two tools that you need to develop apps with Expo: a command line app called Expo CLI to initialize and serve your project and a mobile client app called Expo client to open it on iOS and Android. Any web browser will work for opening the project on the web.
 
-Expo CLI is a tool for developing apps with Expo. In addition to the command-line interface (CLI) it also has a web-based graphical user interface (GUI) that pops up in your web browser when you start your project &mdash; you can use this if you're not yet comfortable with the using terminal or just prefer GUIs, both have similar capabilities.
+## 1. Expo CLI
 
-### Pre-requisities
+Expo CLI is a tool for developing apps and websites with React. Expo CLI also has a web-based GUI that pops up in your web browser when you start your project &mdash; you can use the GUI instead of the command line interface if you're not yet comfortable using a terminal or prefer GUIs, both have similar capabilities.
```

Comment:
```suggestion
Expo CLI is a command line app that is the main interface between a developer and Expo tools. Expo CLI also has a web-based GUI that pops up in your web browser when you start your project &mdash; you can use the GUI instead of the command line interface if you're not yet comfortable using a terminal or prefer GUIs, both have similar capabilities.
```